site/content/integrate/apps/deployment/_index.md

```diff
@@ -49,40 +49,71 @@ For a go app the manifest snippet would look like this:
         }
     ]
 }
-``` 
+```
 
-### Deploying third-party apps to AWS
+### Deploying third-party apps to third-party AWS cloud environment
```

Comment:
What is a " third-party AWS cloud environment"?

Reply:
I'm reverting this..  Obviously, it's causing confusion and I don't fully recall the intention.